### ISSUE TITLE
Weird visual indentation bug

### ISSUE THREAD

**Issue by rafaelrinaldi** [calleerlandsson/pick#188]
After yesterday's release I started getting a weird behavior from `pick` when combined with Git commands specifically, where lists are displayed with a weird indentation for some reason.

```
$ git ls | pick
```

![screen shot 2017-04-20 at 11 34 15 am](https://cloud.githubusercontent.com/assets/87162/25239317/af8635b4-25bd-11e7-80a1-86cee9537aa5.png)

```
$ git checkout $(git for-each-ref refs/heads | pick)
```

![screen shot 2017-04-20 at 11 35 59 am](https://cloud.githubusercontent.com/assets/87162/25239417/eb64acbe-25bd-11e7-984b-ad9d1c97bc29.png)

Any ideas on what is happening?

**Comment by mptre** [comment]
Sorry, I managed to introduce a regression. Please give #187 a try and
see if it solves your problem.

**Comment by mptre** [comment]
Should be fixed in the latest release [v1.6.1].

[v1.6.1]: https://github.com/calleerlandsson/pick/releases/tag/v1.6.1

**Comment by rafaelrinaldi** [comment]
@mptre Thank you! I can confirm it's fixed.
